scripts/scenarios/road-to-utopia.ts

```diff
@@ -0,0 +1,10 @@
+import { CardData } from '../../src/game/ContentTypes'
+
+// HACK: This useless import + function is required for this to count as an ES module, so we can run it
+function noop(c: CardData): CardData {
```

Comment:
Is it really? Isn't the export enough to make it an ES module?

Reply:
You're right, thanks! :D